### ISSUE TITLE
Container PostFX Blur broken as of v3.80.1

### ISSUE THREAD

**Issue by vijayrudraraju** [phaserjs/phaser#6997]
## Version

* Phaser Version: v3.87.0

* Operating system: MacOS

* Browser: All

## Description

`container.postFX.addBlur` does not work after v3.60.0.

Originally, I noticed this locally, but it's easiest to confirm with one of the public examples:
https://phaser.io/examples/v3.85.0/fx/blur/view/container-blur

If you switch the version to v3.60.0 using the provided dropdown, it works. Any version after that, it stops working.

Thanks, I am a huge fan of this game engine!

UPDATE: It actually only works in v3.60.0... is this example supposed to work or is this deprecated?

## Example Test Code

https://phaser.io/examples/v3.85.0/fx/blur/view/container-blur

## Additional Information

<img width="1512" alt="Screenshot 2024-12-27 at 10 19 18 PM" src="https://github.com/user-attachments/assets/b9d53101-e7cd-4fd4-9880-92ff381f66a1" />

**Comment by BenjaminDRichards** [comment]
Yes, this example is supposed to work! The parameters should be `.addBlur(1, 1, 1, 0, 0xffffff, 6)`. The original version `.addBlur(1, 0, 0, 0, 0xffffff, 6)` sets the X and Y blur to 0. It looks like v3.60.0 simply ignored the X and Y values, and the quality wasn't great.

Containers are definitely intended to work with Blur and other FX (renamed to Filters in v4).

This example will be updated for Phaser v4.0.0 with the correct values.
